.github/workflows/nightly_dafny.yml

```diff
@@ -32,3 +32,18 @@ jobs:
     uses: ./.github/workflows/test_models_rust_tests.yml
     with:
       dafny: 'nightly-latest'
+
+  cut-issue-on-failure:
+    runs-on: ubuntu-latest
+    needs: [dafny-nightly-verification, dafny-nightly-java, dafny-nightly-net, dafny-nightly-rust]
+    if: ${{ contains(needs.*.result, 'failure') }}
```

Comment:
Is this `if` expression enough to ensure the step executes when the dependencies fail, or is the `always() &&` needed?

Reply:
Good catch! I had that on when testing and then removed it thinking for a moment it was only for testing.